light_node/src/main.rs

```diff
@@ -46,10 +47,10 @@ mod configuration;
 mod identity;
 mod system;
 
-extern crate jemallocator;
+//extern crate jemallocator;
```

Comment:
@mambisi 
what about this jemalloc?
If everything is better without it, just remove the lines and depencencies please,
If it was just for purpose of memory-profiling and we want jemalloc, plese uncomment these lines

Reply:
@bkontur it is probably a good idea to have this enabled/disabled through a feature flag

Reply:
> n

good idea :)

Reply:
Yeah it was for memory profiling, i will enable it